src/ig/IGExporter.ts

```diff
@@ -475,6 +509,13 @@ export class IGExporter {
           );
         }
       }
+      if (invalidFile) {
+        let message = `Invalid file detected in directory ${dirPath}.`;
+        if (xmlFile) {
+          message += ' XML format not supported, input FHIR definitions must be JSON.';
+        }
```

Comment:
Maybe you should add an `else` here that adds `Input FHIR definitions must be JSON.` to the message?  Otherwise `Invalid file detected in directory` is not a very helpful message (e.g. "why is it invalid?").

Reply:
Improved here https://github.com/FHIR/sushi/pull/262/commits/7c3f17e30520a5f45ada23ce0b7e8f867904ed0f